src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -417,18 +408,18 @@
       "type": "parameter",
       "dataType": "bool",
       "defaultValue": "false",
-      "description": "Whether or not to use Central Package Management - package versions are via central Directory.Packages.props file"
+      "description": "Use Central Package Management where package versions are via central Directory.Packages.props file"
     },
     "vscode": {
-      "displayName": "WebAssembly - Visual Studio Code Debugging",
+      "displayName": "Visual Studio Code Debugging",
       "type": "parameter",
       "dataType": "bool",
       "defaultValue": "true",
-      "description": "Adds the Visual Studio Code Debugging support files for WebAssembly"
+      "description": "Adds the Visual Studio Code Debugging support files for WebAssembly and Skia"
```

Comment:
As the tasks.json and launch.json only have something for WASM and Skia GTK, we should either:
1. Change this to specify Skia Gtk instead of just 'Skia'
2. Update the .vscode files to "support" the other Skia targets

Reply:
Updated to Skia Gtk